config.go

```diff
@@ -97,6 +99,8 @@ func getConfig() *types.Configuration {
 	v.SetDefault("Alertmanager.CheckCert", true)
 	v.SetDefault("Alertmanager.Endpoint", "/api/v1/alerts")
 	v.SetDefault("Alertmanager.ExpiresAfter", 0)
+	v.SetDefault("Alertmanager.DropEventDefaultPriority", "critical")
+	v.SetDefault("Alertmanager.DropEventThresholdList", `[{"priority":"critical", "value":10000}, {"priority":"critical", "value":1000}, {"priority":"critical", "value":100} ,{"priority":"warning", "value":10}, {"priority":"warning", "value":1}]`)
```

Comment:
why don't you use a map[string]int?

Reply:
I use a a list of object to be able to order the thresholds, so I don´t have to import a ordered map lib

Reply:
I changed the way I pare this, by giving the variable DropEventThresholds in the config or the env var ALERTMANAGE_DROPEVENTTHRESHOLDS as a string of comma separated priority:value.
The used var `DropEventThresholdList` is build from this value